test/expected.json

```diff
@@ -3,7 +3,7 @@ Resources:
     Type: AWS::S3::Bucket
     DeletionPolicy: Retain
     Metadata:
-      aws:cdk:path: delivlib-test/CodeCommitPipeline/BuildPipeline/ArtifactsBucket/Resource
```

Comment:
revert ?

Reply:
We can revert after 7 days - we have to wait for the secrets to expire - secret manager has a terrible development experience because you can't delete secrets immediately. I chose to check in an `expected.json` file that is compatible with `npm test update` since I ran into a bunch of problems. But maybe that was just because I was confused (user error) by the duplicate pipelines and won't happen again now that I've experienced the pain. Happy with whichever, let me know.